.github/workflows/run-build-and-acceptance-tests.yml

```diff
@@ -124,8 +127,14 @@ jobs:
       - name: Lint .NET
         run: |
           cd sdk/dotnet && make lint
-  build_and_test:
-    name: Build & Test
+
+  build:
+    name: Build
+    uses: pulumi/pulumi/.github/workflows/build.yml@487cbf87b26496dc7f205eb85912689a36830d6c
```

Comment:
Why's this pinned to a specific commit?

Reply:
It could be @master but then PR wouldn't work because it's not merged yet.